src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -280,12 +281,23 @@ export default class StandardRouteWaypointModel extends BaseModel {
      * Find the matching fix from the `FixCollection` and clone its `StaticPositionModel` this `_positionModel`
      *
      * @for StandardRouteWaypointModel
-     * @method _clonePoisitonFromFix
+     * @method clonePositionFromFix
      * @param fixCollection {FixCollection}
      * @private
      */
-    clonePoisitonFromFix() {
-        const fixModel = FixCollection.findFixByName(this.name);
+    clonePositionFromFix() {
+        if (this.name[0] === VECTOR_WAYPOINT_PREFIX) {
```

Comment:
Why are we using `this.name[0]`?  Name should be a string not an array.

(But I'm on my phone and can't check easily)